src/main.js

```diff
@@ -25,9 +25,7 @@ const { getUserDataPath } = require('./vs/platform/environment/node/userDataPath
 const product = require('../product.json');
 const { app, protocol, crashReporter } = require('electron');
 
-// Disable render process reuse, we still have
-// non-context aware native modules in the renderer.
-app.allowRendererProcessReuse = false;
+app.allowRendererProcessReuse = true;
```

Comment:
Shouldn't this remain `false` for now and only turn into `true` when the user has opted in via `enable-render-process-reuse` below?

Reply:
Changed feature to opt-in.